### ISSUE TITLE
node_id limit question?

### ISSUE THREAD

**Issue by samsonsite1** [MiviaLab/vf3lib#18]
Hello,
I'm looking at both VF2 and VF3 libraries, and noticed VF2 node_id is limited to unsigned short, while VF3 is limited to unsigned int.

Is it ok to change VF2 node_id to unsigned int ?

Just out of curiosity, why was node_id limited to unsigned short?  
Was this a memory concern or a speed concern?

// VF2
typedef unsigned short node_id;
const node_id NULL_NODE=0xFFFF;

// VF3
	typedef uint32_t nodeID_t; /**<Type for the id of the nodes in the graph */
	const nodeID_t NULL_NODE = (std::numeric_limits<uint32_t>::max)();	/**<Null node value */

**Comment by vincarlet** [comment]
Yes, this is mostly related to the size of graphs that were available when vf2 was firstly released.
You can extend the node type from short to int.
